src/implementations/junoswap/helpers.rs

```diff
@@ -227,6 +229,13 @@ pub(crate) struct JunoProvideLiquidityInfo {
 /// ### Returns
 /// The amount of token1 and token2 that should be sent to Junoswap to provide
 /// liquidity and the expected amount of lp tokens to be minted.
+/// 
+/// ### Requirements
+/// - pool_info.token2_reserve > 0
+/// - token2.amount > 0
```

Comment:
Not sure what this means. Can you please either remove this or explain it further so its understandable?